bin/build

```diff
@@ -28,9 +44,20 @@ cp -R /opt/home/.meteor $HOME
 # --json || --debug
 
 cd $INPUT_DIR
-printf "Using $(meteor --version).\n"
+METEOR_VERSION=$(cat $HOME/.meteor/version)
```

Comment:
This is not the same `$HOME` as in [bootstrap.sh:67](https://github.com/onmodulus/docker-build-meteor/pull/12/files#diff-3af436e6760a0c5f1e3d16a3384853a8R67)

Is that an issue?